lost/src/main/java/com/mapzen/android/lost/api/Status.java

```diff
@@ -67,11 +71,10 @@ public Status(int statusCode, PendingIntent pendingIntent) {
    * @param requestCode associated with activity.
    * @throws IntentSender.SendIntentException
    */
-  public void startResolutionForResult(Activity activity, int requestCode)
+  public void startResolutionForResult(final Activity activity, final int requestCode)
```

Comment:
Per comment by @westnordost we should update the Javadoc here to specifically call out the difference with play services in that currently `onActivityResult(...)` will not be invoked for the given `activity` parameter with `RESULT_OK`.

Another option would be to wrap the `SettingsDialogFragment` in its own `Activity` which could perform an additional location settings check when the user returns from the system settings menu. Then if the settings have been updated by the user it could finish itself with `RESULT_OK`.

Of course then we would also need a way to pass the original `LocationSettingsRequest` into the wrapper `Activity` so it could verify location settings requirements have not been met.